src/python/py_imagebufalgo.cpp

```diff
@@ -2795,17 +2800,19 @@ declare_imagebufalgo(py::module& m)
 
         .def_static("resample", &IBA_resample, "dst"_a, "src"_a,
                     "interpolate"_a = true, "roi"_a = ROI::All(),
-                    "nthreads"_a = 0)
+                    "wrapname"_a = "default", "nthreads"_a = 0)
```

Comment:
To match the C++ interface, let's call this parameter "wrap" as well (and the same comment as before about the parameter order).